g2p/tests/public/data/haa.csv

```diff
@@ -0,0 +1,12 @@
+haa,haa-ipa,tth’ą̂ą̈̌,tθ'æ̃ɑ̃
+haa,haa-ipa,nhdhthtthddhwhshzhchch’yhkhghh'
+haa,haa-ipa,̥ndθtθʰtθ̥wʃʒtʃʰtʃ'jxɣhʔ
```

Comment:
These two test cases only have 3 fields each, there's a comma missing between the input and the reference.

Reply:
fixed!